components/profile/rating/src/index.js

```diff
@@ -0,0 +1,92 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+
+class ProfileRating extends Component {
+  constructor (props) {
```

Comment:
Init the state like this:
```
state = { ratingFilledWidth: 0 }
```

Reply:
Is it a convention? Is more clean for me to have it as a constructor declaration but I don't have any problem to change it.